src/components/shared/Button/PaneToggle.css

```diff
@@ -1,18 +1,21 @@
 .toggle-button-start,
 .toggle-button-end {
-  position: absolute;
-  width: 16px;
-  height: 16px;
   transition: transform 0.25s ease-in-out;
-  margin: 0 4px;
   cursor: pointer;
+  align-self: flex-end;
```

Comment:
I don't know if we want to have this on the `toggle-button-start` too, what do you think ?